frontend/src/MafiaApp.js

```diff
@@ -1,33 +1,18 @@
-import React, { useContext } from "react";
+import React from "react";
 
 import WelcomeScreen from "./screens/WelcomeScreen";
 import WaitingRoom from "./screens/WaitingRoom";
-import { MyContext } from "./infra/MyContext";
-import { PHASE } from "./shared_code/consts.js";
+import { Switch, Route } from "react-router-dom";
 
 const MafiaApp = () => {
```

Comment:
is this the convention? not something like AppRoutes?

Reply:
AppRoutes sounds much better
9499074